docs/build/basics/events.md

```diff
@@ -57,27 +57,48 @@ The first `A` means the event is originating from a contract, which will always
 
 There is an unlimited amount of events that can be defined on Flow, but you should know about the most common ones. 
 
-### FLOW Token Events
+### Fungible Token Events
 
-The FLOW Token contract uses the [fungible token standard on Flow](../../build/core-contracts/03-flow-token.md) and is the contract that issues a core FLOW token. As with any contract, it can emit events when interacted with. When we transfer the FLOW token, events are emitted. You can find a lot of details on the events emitted in the [FLOW Token documentation](../../build/core-contracts/03-flow-token.md). 
+All fungible token contracts, including [The FLOW Token contract](../../build/core-contracts/03-flow-token.md),
+use the [fungible token standard on Flow](../../build/core-contracts/02-fungible-token.md).
+As with any contract, the standard emits events when interacted with.
+When any fungible token is transferred, standard events are emitted.
+You can find a lot of details on the events emitted in the [Fungible Token documentation](../../build/core-contracts/02-fungible-token.md). 
 
-The most common events are when tokens are transferred which is accomplished with two actions: withdrawing tokens from the payer and depositing tokens in the receiver. Each of those action has a corresponding event:
+The most common events are when tokens are transferred which is accomplished with two actions: withdrawing tokens from the payer and depositing tokens in the receiver. Each of those actions has a corresponding event:
 
 **Withdraw Tokens**
 
-Event name: `TokensWithdrawn`
+Event name: `FungibleToken.Withdrawn`
+```cadence
+event Withdrawn(type: String,
+                amount: UFix64,
+                from: Address?,
+                fromUUID: UInt64,
+                withdrawnUUID: UInt64,
+                balanceAfter: UFix64)
+```
 
-Mainnet event: `A.1654653399040a61.FlowToken.TokensWithdrawn`
+Mainnet event: `A.0xf233dcee88fe0abe.FungibleToken.Withdrawn`
```

Comment:
I believe identifiers omit the `0x` prefix. Applies in several places below where identifiers are listed